src/Uno.UI/UI/Xaml/Markup/Reader/XamlObjectBuilder.cs

```diff
@@ -725,12 +726,19 @@ private Binding BuildBindingExpression(object? instance, XamlMemberDefinition me
 				binding.RelativeSource = RelativeSource.TemplatedParent;
 			}
 
-			if(bindingNode == null && templateBindingNode == null)
+			if (xBindNode != null)
 			{
-				throw new InvalidOperationException("Unable to find Binding or TemplateBinding node");
+				binding.Source = rootInstance;
+				// TODO: here we should be setting Mode to OneTime by default, and we should also respect x:DefaultBindMode values set 
+				// further up in the tree.
 			}
 
-			foreach (var bindingProperty in (bindingNode ?? templateBindingNode)!.Members)
+			if(bindingNode == null && templateBindingNode == null && xBindNode == null)
```

Comment:
nit: missing space after `if`. Shouldn't some automated checks catch these things?

Reply:
I don't think we run linting on the ci.